### ISSUE TITLE
 Problem to write in Arabic as I noticed! (since QupZilla 1.8.8 and 1.8.9 is out)

### ISSUE THREAD

**Issue by Gamezertruth** [QupZilla/qupzilla#1773]
First I want to say that your browser is more than wonderful! Light on memory and is the best of all browsers out there! I really liked it!

Well I am having a critical problem since QupZilla 1.8.8 and 1.8.9 is out! Which I can not write in Arabic when I play online games ? so my Basic language is the Arabic and the newest version of your browser had a problem to writing in Arabic and I have make a Small test with your browser and still unable to writing in Arabic which I used to chat-chat with the other .... anyway I only able to writing in Arabic when hover my mouse to QupZilla search bar ! and when I hover my mouse on some area of a webpages to chat/ or even trying to typing in Arabic but still the same issue or a bug ! and I kept the older version of QupZilla setup files on my system but i have removed them all by 
Mistake !

note : before I removed them all I was already revert back to old QupZilla 1.8.6 and is fine : D A)

**Comment by JHooverman** [comment]
@srazi: Are you able to reproduce this issue?

**Comment by Gamezertruth** [comment]
no , I could not reproduce this issue ! but all I can say is the The previous version of the program (QupZilla 1.8.6) was working fine and I was able to write in Arabic and English whiteout any issues !

**Comment by nowrep** [comment]
Maybe missing codecs in bundled Qt libraries.

**Comment by Gamezertruth** [comment]
yes I use Windows 7 operating system windows 7 ultimate 32 bit ! and you may need to re-checked your code ! thanks !

**Comment by Gamezertruth** [comment]
any update on my issue ? still waiting for a reply

**Comment by Gamezertruth** [comment]
I was able to reproduce this issue today ! and that it was a bug in your web browser ! 
anyway you need to test Qupzilla with a flash website such as a gaming website ! (btw http://gamezer.com/billiards/ )

**Comment by srazi** [comment]
I can reproduce it using version 1.8.9 but I can't reproduce it using v2.0.0 and v1.8.4. (all downloaded from site)
I tested it using http://www.tizag.com/pics/flash/form1easy.swf under Win10_x64
